src/tiled/invertyaxishelper.h

```diff
@@ -24,32 +24,41 @@
 #include "preferences.h"
 #include "mapdocumentactionhandler.h"
 
-class InvertYAxisHelper{
-    public:
-        InvertYAxisHelper() {}
-        ~InvertYAxisHelper() {}
-        
-        // Inverts Y coordinate in grid
-        float getY(float y) const
+namespace Tiled{
+    namespace Internal {
+
+        class InvertYAxisHelper
         {
-            // Check if Invert Y Axis is set
-            if(Tiled::Internal::Preferences::instance()->invertYAxis())
+        public:
+            // Constructors
+            InvertYAxisHelper()  {}
```

Comment:
You should drop this constructor.

Reply:
I maintained it to be able to use the class without providing a Map, like I'm doing at the tileY method. Should I give a map anyway?

Reply:
Yes, because actually `tileY` can't be implemented correctly without a map, and currently its gets the map from the `MapDocumentActionHandler`, which is not always referring to the right map.